src/lib.rs

```diff
@@ -298,7 +303,7 @@ impl Message {
     /// ```ignore
     /// let signer: Vec<u8> = message.verify_eip191(&signature)?;
     /// ```
-    pub fn verify_eip191(&self, sig: &[u8; 65]) -> Result<Vec<u8>, VerificationError> {
```

Comment:
What's the motivation to remove the length constraint?

Reply:
EIP1271 allows signatures of any length unfortunately